ajax/showfields.php

```diff
@@ -31,6 +31,39 @@
 
 include ('../../../inc/includes.php');
 
+$criteria = [
```

Comment:
the below code is good, but i think it could be better in a dedicated function (can be at least unit tested)

Reply:
I agree : this is a copy of code found in front/formdisplay.php with modifications dedicated to use with AJAX.

As I wand to avoid a regression, I would defer factorisation in the develop branch.